### ISSUE TITLE
Receiving 'nil' for apiRequestsCount and responseTime

### ISSUE THREAD

**Issue by landesbergn** [sxg/ForecastIO#54]
Everything about the API and ForecastIO is working well, except having trouble with getting the metadata for the **apiRequestsCount** and **responseTime**. Not sure if the issue is in my code, in ForecastIO, or in the DarkSky API

My code:
```
self.client.getForecast(latitude: lat, longitude: lon, excludeFields: [.alerts, .daily, .flags, .minutely]) { result in
            switch result {
            case .success(let currentForecast, let requestMetadata):
                //  We got the current forecast!
                if let currentFeelsTemperature: Double = currentForecast.currently?.temperature {
                    print(currentFeelsTemperature)
                    DispatchQueue.main.async {
                        self.feelsLikeTmpLbl.text = String(Int(round(currentFeelsTemperature)))
                    }
                }
                if let summaryInfo: String = currentForecast.currently?.summary {
                    print(summaryInfo)
                    DispatchQueue.main.async {
                        self.summaryLbl.text = summaryInfo
                    }
                }
                print(requestMetadata)
            case .failure(let error):
                //  Uh-oh. We have an error!
                print(error)
            }
        }
```

Returns:
```
RequestMetadata(cacheControl: Optional("max-age=3600"), apiRequestsCount: nil, responseTime: nil)
```

**Comment by sxg** [comment]
That's strange—the unit tests for `RequestMetadata` look good to me. I'll take a closer look at this soon. I'll be busy early this week, but I'll try to take a look this weekend and update you on the issue.

**Comment by sxg** [comment]
Hey, sorry for the delay! I found some time for this issue, and the fix is now available in ForecastIO 5.1.0. It looks like the headers require lowercase strings, which is why you weren't getting the metadata. Sorry about that and the delay! Let me know if you still have any problems with this.

**Comment by landesbergn** [comment]
Thanks for the update! Everything seems to be working great now.
